bin/pgenv

```diff
@@ -517,6 +519,31 @@ pgenv_configuration_load(){
     PGENV_WARNINGS=true  # for backward compatibility
 }
 
+# Function to merge older string-based configuration variables into their array
+# replacements. Arrays are the preferred way to configure options, since they're
+# better at paramter passing, but we need to support older configurations that
+# used string variables.
+pgenv_merge_arrays() {
+    if [ ! -z ${PGENV_INITDB_OPTS+x} ]; then
```

Comment:
`! -z ${PGENV_INITDB_OPTS+x}` tests if the variable has been set at all, since some folks may have it empty so as to disable all the opts. See [this SO answer](https://stackoverflow.com/a/13864829/79202) for details.